AlarmSources/Fax/FaxAlarmSource.cs

```diff
@@ -132,6 +133,39 @@ private void EnsureDirectoriesExist()
             }
         }
 
+        private void ProcessNewPdf(FileInfo file)
+        {
+            string tiffFilePath = Path.Combine(file.DirectoryName, "convert.tif");
+            Logger.Instance.LogFormat(LogType.Trace, this, Properties.Resources.GhostscriptConvert, file.FullName);
+
+            try {
+                //Convert to tiff
+                GhostscriptImageDevice dev = new GhostscriptImageDevice();
+                dev.Device = "tiffgray";
+                dev.GraphicsAlphaBits = GhostscriptImageDeviceAlphaBits.V_4;
+                dev.TextAlphaBits = GhostscriptImageDeviceAlphaBits.V_4;
+                dev.Resolution = 300;
+                dev.InputFiles.Add(file.FullName);
+                dev.OutputPath = tiffFilePath;
+                dev.Process();
+            } catch(GhostscriptException ex)
+            {
+                Logger.Instance.LogFormat(LogType.Error, this, Properties.Resources.GhostscriptConvertError);
+                Logger.Instance.LogException(this, ex);
+
+                string archivedFilePath = Path.Combine(_archivePath.FullName, file.Name);
+                MoveFileTo(file, archivedFilePath);
+
+                return;
```

Comment:
Das "return" müsste hier nicht sein, wenn man die beiden unteren Zeilen in das try{} verschiebt.

Weiterhin bitte unnötige Kommentare vermeiden (die beiden unten).